### PR TITLE
Autotest: check for empty message before trying to use it

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

**Review by amilcarlucas — APPROVED** [review]
good catch. LGTM

File: Tools/autotest/common.py [hunk 1]
```diff
@@ -3632,10 +3632,10 @@ def get_parameter_direct(self, name, attempts=1, timeout=60, verbose=True, timeo
                 if delta_time > timeout:
                     break
                 m = self.mav.recv_match(type='PARAM_VALUE', blocking=True, timeout=0.1)
-                if verbose:
```
Comment (peterbarker) [inline Tools/autotest/common.py:3635]: This one was entirely intentional.  I'd prefer to keep it for the time being.

**Review by peterbarker — CHANGES_REQUESTED** [review]
Looks good apart from one I'd like to keep for the moment.

**Comment by khancyr** [comment]
change done !

**Review by peterbarker — APPROVED** [review]

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**